src/Tools/DynamoInstallDetective/ProductLookUp.cs

```diff
@@ -299,12 +299,18 @@ public IInstalledProduct GetLatestProduct()
             return Products.LastOrDefault();
         }
 
-        public virtual void LookUpAndInitProducts(IProductLookUp lookUp)
+        public virtual void LookUpAndAddProducts(IProductLookUp lookUp)
```

Comment:
Can't change this name - it's public.

Reply:
I will revert the change, the original name is not ideal but should be good enough since I updated the function description